lib/warbler/traits/bundler.rb

```diff
@@ -45,7 +45,7 @@ def add_bundler_gems; require 'bundler'
             while ! full_gem_path.join('bundler.gemspec').exist?
               full_gem_path = full_gem_path.dirname
               # if at top of the path, meaning we cannot find bundler.gemspec, abort.
-              if full_gem_path.to_s =~ /^[\.\/]$/
+              if full_gem_path.root?
```

Comment:
The Pathname#root? means "does it consist of consecutive slashes?", according to https://rubyapi.org/3.4/o/pathname#method-i-root-3F

This code _seems_ to ask also "is the full_gem_path the current directory? (aka the single dot)". Do you agree with that?

Reply:
```suggestion
              if full_gem_path.root? || full_gem_path.to_s == '.'
```

Reply:
I'm fine with that change, although the code probably wouldnt be in the while loop if bundler.gemspec was found in "."